### DISCUSSION TITLE
SL-ANIMALS-DVS

### DISCUSSION THREAD

**Discussion by fabrizio-ottati** [neuromorphs/tonic#208] [Ideas]
Hi everyone. I saw this article about a [new DVS dataset](http://www2.imse-cnm.csic.es/neuromorphs/index.php/SL-ANIMALS-DVS-Database) in which umans emulate the sign language of animals in front of a DVS camera. Should we add it to Tonic?

I see that the file format is AEDAT; hence, we should ask the authors if they can provide the data in NumPy or HDF5 format.

**Comment by biphasic** [comment]
if you're going to use it yes then I'm happy to look at a PR. We can maybe do a dynamic import of AEDAT https://github.com/neuromorphicsystems/aedat until authors upload an hdf5 version of their dataset. 

Just a word of caution here. If you're not going to use it, it might not be worth your time, because there is DVS gesture and ASL-DVS already.
